compiler/resolution/functionResolution.cpp

```diff
@@ -1622,23 +1622,20 @@ moreSpecific(FnSymbol* fn, Type* actualType, Type* formalType) {
 
 static bool
 computeActualFormalAlignment(FnSymbol* fn,
-                             Vec<Symbol*>& formalIdxToActual,
-                             Vec<ArgSymbol*>& actualIdxToFormal,
+                             std::map<int, Symbol*>& formalIdxToActual,
+                             std::map<int, ArgSymbol*>& actualIdxToFormal,
                              CallInfo& info) {
```

Comment:
I may be missing something but it seems to me that the actual "info" could be const CallInfo& info.